sklearn/compose/_column_transformer.py

```diff
@@ -85,12 +85,11 @@ class ColumnTransformer(_BaseComposition, TransformerMixin):
         estimator must support `fit` and `transform`.
 
     sparse_threshold : float, default = 0.3
-        If the transformed output consists of a mix of sparse and dense data,
-        it will be stacked as a sparse matrix if the density is lower than this
-        value. Use ``sparse_threshold=0`` to always return dense.
-        When the transformed output consists of all sparse or all dense data,
-        the stacked result will be sparse or dense, respectively, and this
-        keyword will be ignored.
+        If the transformed output contains sparse matrices, it will be stacked
```

Comment:
Would "If the output of the different transformers contains sparse matrices, ..." be clearer? Because now if I start reading this, I would interpret "transformed output" as the output of the ColumnTransformer (so already stacked), and then the rest of the sentence reads a bit strange.

Reply:
yeah I wasn't happy with that wording either. I like yours.